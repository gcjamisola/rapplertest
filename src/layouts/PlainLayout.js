import React, { Component } from 'react';
import PropTypes from 'prop-types';

import Header from '../components/Header';
import Footer from '../components/Footer';

import { Wrapper } from '../styles';

class PlainLayout extends Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <Wrapper>
                <Header/>
                { this.props.children }
                <Footer/>
            </Wrapper>
        )
    }
}

PlainLayout.propTypes = {
    children: PropTypes.node
}

export {PlainLayout};