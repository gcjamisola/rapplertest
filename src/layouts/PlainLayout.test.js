import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { PlainLayout } from './PlainLayout';

describe('<PlainLayout/>', () => {
    it('renders <PlainLayout/> component', () => {
        const component = shallow(<PlainLayout/>);
        const tree = toJson(component);
        expect(tree).toMatchSnapshot();
    });
});
