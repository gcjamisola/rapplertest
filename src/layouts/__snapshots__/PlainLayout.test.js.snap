// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlainLayout/> renders <PlainLayout/> component 1`] = `
<styled.div>
  <Header />
  <Footer />
</styled.div>
`;
